apps/frontend/src/components/global/UploadNexus.vue

```diff
@@ -108,7 +108,7 @@ const localTab = new LocalStorageVal<string>('nexus_curr_tab');
 export default class UploadNexus extends mixins(ServerMixin, RouteMixin) {
   @Prop({default: true}) readonly visible!: boolean;
   @Prop({default: false}) readonly persistent!: boolean;
-  activeTab: string = localTab.get_default('uploadtab-local');
+  activeTab: string = localTab.getDefault('uploadtab-local');
```

Comment:
ideally we have cypress tests to validate these changes, but it's unfortunately not the case.  can you attest that you've spun up a heimdall instance and done manual testing on the ui to hit the various components that were modified?